presto-tests/src/main/java/io/prestosql/tests/DistributedQueryRunner.java

```diff
@@ -467,7 +467,7 @@ private static void closeUnchecked(AutoCloseable closeable)
         private String environment = ENVIRONMENT;
         private Optional<Path> baseDataDir = Optional.empty();
 
-        protected Builder(Session defaultSession)
+        public Builder(Session defaultSession)
```

Comment:
you can use `io.prestosql.tests.DistributedQueryRunner.Builder#amendSession` instead